plugin.xml

```diff
@@ -39,6 +42,14 @@
         <header-file src="src/ios/SegmentCordovaPlugin.h" target-dir="SegmentCordovaPlugin"/>
         <source-file src="src/ios/SegmentCordovaPlugin.m" target-dir="SegmentCordovaPlugin"/>
 
-        <framework src="Analytics" type="podspec" spec="3.7.1"/>
+        <podspec>
+            <config>
+                <source url="https://cdn.cocoapods.org/"/>
+            </config>
+            <pods>
+                <pod name="Analytics" spec="3.7.1"/>
+                <pod name="Segment-Appboy" spec="3.0.0"/>
+            </pods>
+        </podspec>
```

Comment:
What is this change doing?

Reply:
2 things.

1. Use the "new" [podspec](https://cordova.apache.org/docs/en/latest/plugin_ref/spec.html#podspec-ios) element to replace the deprecated usage of `framework` with `type=podspec`
1. Add a dependency on the `Segment-Appboy` Cocoapod.  Appboy is the what Braze used to be called.